src/main/java/org/openmrs/addonindex/rest/AddOnController.java

```diff
@@ -40,6 +40,11 @@ public AddOnController(Index index) {
 	                                           @RequestParam(value = "q", required = false) String query) throws Exception {
 		return index.search(type, query);
 	}
+
+	@RequestMapping(method = RequestMethod.GET, value = "/api/v1/tags")
```

Comment:
In fact I would just handle this as a third optional parameter in the search method in this controller. And you'd add this as a third argument to the index.search method.